ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main.yaml

```diff
@@ -33,6 +33,18 @@ tests:
     - chain: ipi-gcp-pre
     - chain: workers-scale
     workflow: openshift-qe-ocp-qe-perfscale-ci-tests
+- as: ocp-qe-perfscale-router-perf-gcps
```

Comment:
does the gcps at the end stand for anything here?

Reply:
This is a typo for gcp, I did another update. I tried to add router-perf test with different providers in the config.